### ISSUE TITLE
Frozen the header row in RouteTable

### ISSUE THREAD

**Issue by zachzwy** [trynmaps/metrics-mvp#451]
Frozen the header row in RouteTable, so that users can easily tell what each column means.

## Current:
When you scroll down, you might need to refer back to the top to double-check what each column means.
<img width="1436" alt="Screen Shot 2019-11-24 at 4 36 10 AM" src="https://user-images.githubusercontent.com/22914663/69494742-f86ad400-0e73-11ea-90a4-b720470ca647.png">

## Suggestion:
Frozen the header row

<img width="712" alt="Screen Shot 2019-11-24 at 4 41 01 AM" src="https://user-images.githubusercontent.com/22914663/69494799-a1193380-0e74-11ea-97f2-14b846dbda01.png">

**Comment by hathix** [comment]
Good idea. Could use https://material-ui.com/components/tables/#fixed-header.
